environment.yml

```diff
@@ -0,0 +1,7 @@
+name: similar-words
+channels:
+    - default
+dependencies:
+    - python=3.6
+    - jupyter
```

Comment:
Lets make jupyter a dev-dependency

Reply:
by dev dependency do you mean developer's choosing i.e. don't include it in the yml file? It is only there for developer convenience so we could just leave it out; makes more sense.